cli/src/index.ts

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env node
+
+import type { DecodeOptions, EncodeOptions } from '@byjohann/toon'
+import process from 'node:process'
+import { DELIMITERS } from '@byjohann/toon'
+import { Command } from 'commander'
+import { z } from 'zod'
+import { decodeToJson } from './decode.js'
+import { encodeToToon } from './encode.js'
+
+interface CliOptions {
```

Comment:
maybe use mode = Enum(encode, decode) instead of booleans so that any future usecase of modes is taken into consideration?